aten/src/ATen/core/function_schema_inl.h

```diff
@@ -29,13 +29,17 @@ inline std::ostream& operator<<(std::ostream& out, const FunctionSchema& schema)
   }
 
   out << ") -> ";
-  if (schema.returns().size() == 1) {
-    out << schema.returns().at(0).type()->str();
-  } else if (schema.returns().size() > 1) {
+
+  const auto& returns = schema.returns();
+  if (schema.is_varret()) {
```

Comment:
if varret is analogous to vararg, it should be appended after the previous args. So it'd be nice to do so for generality

Reply:
sure, let's make it nice :-)